go/ct/spc/specification.go

```diff
@@ -1710,6 +1731,73 @@ func logOp(n int) []Rule {
 	return rules
 }
 
+func nonStaticSelfDestructRules(revision Revision, warm bool, destinationColdCost, accountCreationFee, refundGas vm.Gas) []Rule {
+
+	var targetWarm Condition
+	var warmColdString string
+	if warm {
+		warmColdString = "warm"
+		targetWarm = IsAddressWarm(Param(0))
+	} else {
+		warmColdString = "cold"
+		targetWarm = IsAddressCold(Param(0))
+	}
+
+	name := fmt.Sprintf("_%v_%v", strings.ToLower(revision.String()), warmColdString)
+
+	instruction := instruction{
+		op:        SELFDESTRUCT,
+		name:      name,
+		staticGas: 5000,
+		pops:      1,
+		conditions: []Condition{
+			Eq(ReadOnly(), false),
+			IsRevision(revision),
+			HasNotSelfDestructed(),
```

Comment:
How about cases where the account has self-destructed before? Is there a specification rule for that?